ext/rotoscope/rotoscope.c

```diff
@@ -178,17 +178,17 @@ static bool in_fork(Rotoscope *config)
 
 static bool tracecmp(rs_tracepoint_t *a, rs_tracepoint_t *b)
 {
-  return (!strcmp(a->method_name, b->method_name) &&
-          !strcmp(a->entity, b->entity) &&
+  return (!strcmp(StringValueCStr(a->method_name), StringValueCStr(b->method_name)) &&
+          !strcmp(StringValueCStr(a->entity), StringValueCStr(b->entity)) &&
```

Comment:
Couldn't you use rb_str_cmp here?

```
  return (!rb_str_cmp(a->method_name, b->method_name) &&
          !rb_str_cmp(a->entity, b->entity) &&
```

not only does that simplify the code, but it should be faster too.